src/orion/benchmark/benchmark_client.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.benchmark.client` -- Benchmark client
+================================================================
+
+.. module:: client
+   :platform: Unix
+   :synopsis: Client to use Orion benchmark.
+
+"""
+import datetime
+
+from orion.benchmark import Benchmark, Study
+from orion.benchmark.assessment.base import BenchmarkAssessment
+from orion.benchmark.task.base import BenchmarkTask
+from orion.core.utils.exceptions import NoConfigurationError
+from orion.storage.base import get_storage, setup_storage
+
+
+def get_or_create_benchmark(
```

Comment:
We can polish this in another PR, but it would be great to have a consistent interface with the ExperimentClient. We now have [`build_experiment`](https://github.com/Epistimio/orion/blob/develop/src/orion/client/__init__.py#L44) to create executable experiments and [`get_experiment`](https://github.com/Epistimio/orion/blob/develop/src/orion/client/__init__.py#L246) to load a readable/writable experiment.

Reply:
okay.  We can think about this a little bit later. I was actually trying to avoid the too much concepts like `experiment` has ( `we get experimentclient, experimentbulder, experimetview and experiment `). It may be necessary and understandable for `experiment` as it is the key code of Orion :)

Reply:
I agree there is a lot, too much, for experiment. Do you think `get_experiment/benchmark` and `build_experiment/benchmark` could be enough? As you say, `get` is for reading the states or perhaps minor db edits like updating trial status, and `build` would be for fully loading experiment/benchmark object in order to execute them. They main difference is that `get` only load the data while `build` needs to make sure we are running the same config.

Reply:
Agree, we can polish and make it consistent later, when we add the get all benchmarks interface maybe :)

Reply:
Sounds good :)